sys/include/net/ieee802154/radio.h

```diff
@@ -484,6 +488,18 @@ struct ieee802154_radio_ops {
      * @pre call to @ref ieee802154_radio_ops::request_on was successful.
      *
      * @post the transceiver state is @ref IEEE802154_TRX_STATE_TRX_OFF
+     * During boot or in case the radio doesn't support @ref
+     * IEEE802154_CAP_REG_RETENTION when @ref off was called, the
+     * Physical Information Base will be undefined. Thus, for take into
```

Comment:
```suggestion
     * Physical Information Base will be undefined. Thus, take into
```

Either there is a word missing or one too much ;)

Reply:
done!

Reply:
Please squash!